projects/igniteui-angular/src/lib/date-picker/date-picker.component.html

```diff
@@ -1,54 +1,30 @@
-<ng-template #labelTemplate>
-    <ng-content select="[igxLabel]"></ng-content>
-</ng-template>
+<igx-input-group [suppressInputAutofocus]="true">
+    <igx-prefix *ngIf="!this.toggleComponents.length" (click)="this.toggle()">
+        <igx-icon [title]="this.value ? 'Change Date' : 'Choose Date'">today</igx-icon>
+    </igx-prefix>
 
-<ng-template #readOnlyDatePickerTemplate>
-    <igx-input-group (click)="openDialog()">
-        <ng-container ngProjectAs="[igxLabel]" *ngTemplateOutlet="labelTemplate"></ng-container>
-        <igx-prefix>
-            <igx-icon>today</igx-icon>
-        </igx-prefix>
-        <label *ngIf="labelVisibility&&!_labelDirectiveUserTemplate" igxLabel>{{labelInternal}}</label>
-        <input
-            class="igx-date-picker__input-date"
-            igxInput
-            [value]="displayData || ''"
-            [disabled]="disabled"
-            (blur)="onBlur($event)"
-            [tabindex]='editorTabIndex'
-            readonly
-        />
-    </igx-input-group>
-</ng-template>
+    <input class="igx-date-picker__input-date" igxInput [igxDateTimeEditor]="this.inputFormat"
+        [minValue]="this.minValue" [maxValue]="this.maxValue" [spinDelta]="this.spinDelta" [spinLoop]="this.spinLoop"
+        [displayFormat]="this.applyCustomFormat()" [disabled]="this.disabled" [placeholder]="this.placeholder"
+        [readonly]="!this.isDropdown" [igxTextSelection]="this.isDropdown" [required]="this.required"
```

Comment:
IIRC, we were avoiding setting the native `required` on inputs in other editors, because there's no ngControl on them and the `igxInput` directive will try and validate using the native call, which may collide with the parent component model state. Worth looking into